.github/workflows/test.yaml

```
@@ -101,13 +101,13 @@ jobs:
           k3s-channel: "${{ matrix.kubernetes }}"
           prefetch-timeout: 5m
           prefetch-images: |
-            registry.developers.crunchydata.com/crunchydata/crunchy-pgbackrest:ubi9-2.56.0-2547
-            registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi9-17.7-2547
-            registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi9-16.11-2547
-            registry.developers.crunchydata.com/crunchydata/crunchy-pgbouncer:ubi9-1.24-2547
-            registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi9-0.18.1-2547
-            registry.developers.crunchydata.com/crunchydata/crunchy-upgrade:ubi9-18.1-2547
-            registry.developers.crunchydata.com/crunchydata/crunchy-pgadmin4:ubi9-9.8-2547
+            registry.developers.crunchydata.com/crunchydata/crunchy-pgbackrest:ubi9-2.56.0-2550
+            registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi9-17.7-2550
+            registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi9-16.11-2550
+            registry.developers.crunchydata.com/crunchydata/crunchy-pgbouncer:ubi9-1.25-2550
+            registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi9-0.18.1-2550
+            registry.developers.crunchydata.com/crunchydata/crunchy-upgrade:ubi9-18.1-2550
+            registry.developers.crunchydata.com/crunchydata/crunchy-pgadmin4:ubi9-9.8-2550
```

Again, we should add the operator image to prefetch (as it's used as the collector image):
```suggestion
            registry.developers.crunchydata.com/crunchydata/crunchy-pgadmin4:ubi9-9.8-2550
            registry.developers.crunchydata.com/crunchydata/postgres-operator:ubi9-5.8.6-0
```